### PR TITLE
using regex to reformat the opening attributes for cleaner git diffs

### PR DESCRIPTION
**Pull request by SignpostMarv**
this is the slightly cludgy use of regex I referenced on twitter over the weekend.

### CODE REVIEW

**Comment by muglug** [comment]
Mind fixing the Psalm bug?

**Comment by SignpostMarv** [comment]
@muglug wasn't getting those locally when I posted last night; was getting a bunch of psalm errors on 0e8fa0e73e3e4530ed88d457194689eabec9b905

will have a look later :)

File: src/Psalm/ErrorBaseline.php [hunk 1]
```diff
@@ -261,6 +263,26 @@ function (string $extension) : string {
...
         $baselineDoc->formatOutput = true;
 
-        $fileProvider->setContents($baselineFile, $baselineDoc->saveXML());
+        $xml = preg_replace_callback(
+            '/<files (psalm-version="[^"]+") (?:php-version="(.+)">\n)/',
+            function (array $matches) : string {
+                return
+                    '<files' .
+                    "\n  " .
+                    $matches[1] .
```
Comment (psalm-shepherd) [inline src/Psalm/ErrorBaseline.php]: Right operand cannot be mixed
```
                    $matches[1] .
                    ^^^^^^^^^^^
```

File: src/Psalm/ErrorBaseline.php [hunk 2]
```diff
@@ -261,6 +263,26 @@ function (string $extension) : string {
...
+                    "\n" .
+                    '  php-version="' .
+                    "\n    " .
+                    implode("\n    ", explode('&#10;&#9;', $matches[2])) .
```
Comment (psalm-shepherd) [inline src/Psalm/ErrorBaseline.php]: Argument 2 of explode cannot be mixed, expecting string
```
                    implode("\n    ", explode('&#10;&#9;', $matches[2])) .
                                                           ^^^^^^^^^^^
```

**Comment by psalm-shepherd** [comment]
Psalm found some errors

File: src/Psalm/ErrorBaseline.php (hunk 1 above)
Comment (SignpostMarv) [reply src/Psalm/ErrorBaseline.php]: should be resolved by 7de56af, but is impacted by #1896

File: src/Psalm/ErrorBaseline.php (hunk 2 above)
Comment (SignpostMarv) [reply src/Psalm/ErrorBaseline.php]: should be resolved by 7de56af, but is impacted by #1896

**Comment by SignpostMarv** [comment]
> Psalm found some errors

the @psalm-shepherd bot comment might be more useful if it gives similar psalm-version info as to the signature present in psalm baseline, i.e. 

> Psalm found some errors (using 3.4.7 @ caf7737bbe5a5e9cbdfcc1ada83ea576ca07c9ff)

**Comment by muglug** [comment]
Mind merging/rebasing master?

**Comment by SignpostMarv** [comment]
> Mind merging/rebasing master?

give me about an hour.

**Comment by SignpostMarv** [comment]
hrm. did I mess up the rebase there?

**Pull request merged**